liubai-frontends/liubai-weixin/miniprogram/utils/liu-util/index.ts

```
@@ -0,0 +1,67 @@
+export class LiuUtil {
+
+  static getCharacteristic() {
+    return handleCharacteristic()
+  }
+
+  static getDeviceString() {
+    return handleDeviceString()
+  }
+
+  static goHome() {
+    const res1 = this.naviBackToHome()
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)